search/search.go

```diff
@@ -41,8 +41,13 @@ func MatchDelimiters(line, flagKey, delimiters string) bool {
 		return true
 	}
 	for _, left := range delimiters {
+		var sb strings.Builder
```

Comment:
Nit: perhaps we could add unit tests for this if there isn't any already?

Reply:
This should be covered by the current unit tests afaik, it's just changing the current implementation details.